ontology/yaml/resources/subfields/subfields.yaml

```diff
@@ -261,6 +261,7 @@ measurement:
   angularmomentum: "The quantity of rotation of a body, which is the product of its moment of inertia and its angular velocity."
   aqi: "Air quality index, a set of special standard gradings for the pollution in air."
   area: "Amount of space within the perimeter of a 2D shape"
+  batterycapacity: "Energy charge in a battery"
```

Comment:
This is not a valid measurement type. Please use `electriccharge`

Reply:
fixed thank you